packages/contracts-bedrock/CONTRIBUTING.md

```diff
@@ -90,16 +91,16 @@ To deploy the smart contracts on a local devnet, run `make devnet-up` in the mon
 
 In order to make sure that we don't accidentally overwrite storage slots, contract storage layouts are checked to make sure spacing is correct.
 
-This uses the `.storage-layout` file to check contract spacing. Run `pnpm validate-spacers` to check the spacing of all contracts.
+This uses the `.storage-layout` file to check contract spacing. Run `just validate-spacers` to check the spacing of all contracts.
 
 #### Gas Snapshots
 
-We use forge's `gas-snapshot` subcommand to produce a gas snapshot for most tests within our suite. CI will check that the gas snapshot has been updated properly when it runs, so make sure to run `pnpm gas-snapshot`!
+We use forge's `gas-snapshot` subcommand to produce a gas snapshot for most tests within our suite. CI will check that the gas snapshot has been updated properly when it runs, so make sure to run `just gas-snapshot`!
```

Comment:
I know this is outside the scope of the PR, but good to use this opportunity to keep it updated.

```suggestion
We use forge's `gas-snapshot` subcommand to produce a gas snapshot for tests in `Benchmark.t.sol`. CI will check that the gas snapshot has been updated properly when it runs, so make sure to run `just gas-snapshot`!
```